provisioning/osx/configure/src/nginx.js

```diff
@@ -0,0 +1,60 @@
+import execa from 'execa';
+import R from 'ramda';
+import fs from 'fs';
+
+const baseConfig = `server {
+  listen 80;
+  location / {
+    return 404;
+  }
+}
+`;
+
+export const create = (services, domain) => `
+${baseConfig}${R.join(
+  '',
+  R.map(
+    s => `
```

Comment:
A little hard to follow this, possibly pull out the template literal into its own function?